cms/articles/models.py

```diff
@@ -269,3 +270,9 @@ def previous_version(self, request: "HttpRequest", version: int) -> "TemplateRes
         )
 
         return response
+
+    @property
+    def topic_ids(self) -> list[str]:
+        """Returns a list of topic IDs associated with the parent article series page."""
+        parent = self.get_parent().specific
+        return list(parent.topics.values_list("topic_id", flat=True)) if hasattr(parent, "topics") else []
```

Comment:
```suggestion
        return list(self.get_parent().specific_deferred.topics.values_list("topic_id", flat=True))
```

1. `specific_deferred` will help not load the whole thing
2. we know the parent has a `topic` attribute

Reply:
Performance improvements and a lazy approach here are awesome.

Done.